CHANGELOG.md

```diff
@@ -35,3 +34,12 @@ following function names:
 
 * Function `Get-CAcl` to get the access control (i.e. security descriptor) for a registry key, file, or directory.
 Supports getting only specific sections/parts of the security descriptor, too. Works across PowerShell editions.
+* Function `Get-CPrivilege` (migrated from Carbon), which gets a user/group's rights/privileges.
+* Function `Grant-CPrivilege` (migrated from Carbon), which grants a user/group rights/privileges.
+* Function `Revoke-CPrivilege` (migrated from Carbon), which removes a user/group's rights/privileges.
+* Function `Test-CPrivilege` (migrated from Carbon), which tests a user/group's rights/privileges.
+* Function `Test-CPrivilegeName`, which tests if rights/privileges are supported on the current operating system.

```

Comment:
I think you're missing `Test-CPermission` from this list?

Reply:
Omitted on purpose. Although the module exports those functions, they are considered internal and should only be used by other Carbon modules.